app/Helpers/Payment/MolliePaymentProvider.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace App\Helpers\Payment;
+
+use \Mockery;
+use App\Helpers\Payment\PaymentProvider;
+use App\Helpers\Payment\PaymentInterface;
+
+class MolliePaymentProvider implements PaymentProvider
+{
+
+    const CURRENCY = "EUR";
```

Comment:
Ik begrijp  niet waar deze constant voor is. De currency wordt toch opgehaald via de PaymentInterface?

Reply:
Dit is een stukje code wat ik de hele tijd heen en weer aan het reworken was, maar deze constant moet er inderdaad uit.